src/Handler/CurlFactory.php

```diff
@@ -419,6 +419,8 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf)
                         !\GuzzleHttp\is_host_in_noproxy($host, $options['proxy']['no'])
                     ) {
                         $conf[CURLOPT_PROXY] = $options['proxy'][$scheme];
+                        if (isset($options['proxy']['username'])&&$options['proxy']['password'])
+                            $conf[CURLOPT_PROXYUSERPWD] = $options['proxy']['username'].':'.$options['proxy']['password'];
```

Comment:
There are some style issues here. Try to follow the style you see on the rest of the file. like adding curly braces, or spaces between `&&`

Reply:
Thank you for your note.
I edited it.